### PR TITLE
Fixing the Coding Style

### PR DESCRIPTION
**Pull request by arcanedev-maroc**
ping @caouecs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 _site
+/.idea/
```
Comment (caouecs) [inline .gitignore]: Please, do an another PR for .gitignore

Comment (arcanedev-maroc) [inline .gitignore]: Ok, ça marche.

**Comment by arcanedev-maroc** [comment]
@caouecs, je vais fermer ce PR et recreer un autre pck il y a un conflit pour le fusionner (I'm too lazy to rebase it :stuck_out_tongue_closed_eyes:).

**Comment by caouecs** [comment]
No need to update .php_cs ?

**Comment by arcanedev-maroc** [comment]
I don't know. Honestly, i've never used the `fabpot/php-cs-fixer` package.
But now, you're using StyleCI, so i don't see why the utility of using this package.

Should we remove .php_cs and `fabpot/php-cs-fixer` completely and using StyleCI instead ??

**Comment by arcanedev-maroc** [comment]
See #470
